.gitignore

```diff
@@ -11,6 +11,7 @@ public
 node_modules/
 .tmuxp.*
 .idea
+/cypress-test
```

Comment:
I don't think this folder is being used anymore, so I don't think we need to add it to the gitignore?

Reply:
This is still being used with the cypress tests for watching styles

Reply:
It is? I can't see where...